tests/ttnn/unit_tests/operations/test_new_conv2d.py

```diff
@@ -530,20 +530,18 @@ def test_conv_features_multi_device(
 @pytest.mark.parametrize(
     "batch_size, output_channels, input_channels, input_height, input_width, filter_height, filter_width, pad_h, pad_w, act_block_w_div",
     (
-        (2, 128, 128, 9, 9, 3, 3, 0, 0, 1),
```

Comment:
any reason why where few test cases removed?

Reply:
They were unnecessary.

Reply:
test coverage for width sharding is miniscule as is, but if you think they are not needed ok